src/blueprints/tv2/action-factories/tv2-dve-action-factory.ts

```diff
@@ -69,32 +73,33 @@ export class Tv2DveActionFactory {
 
   public isDveAction(action: Action): boolean {
     const tv2Action: Tv2PartAction | Tv2PieceAction = action as Tv2PartAction | Tv2PieceAction
-    return [
-      Tv2ActionContentType.DVE_LAYOUT,
-      Tv2ActionContentType.DVE_INSERT_SOURCE_TO_INPUT,
-      Tv2ActionContentType.RECALL_DVE,
-      Tv2ActionContentType.DVE_INSERT_LAST_VIDEO_CLIP_TO_INPUT
-    ].includes(tv2Action.metadata.contentType)
+    const actionSubtype: Tv2ActionSubtype | undefined = tv2Action.metadata.actionSubtype
+    return actionSubtype !== undefined && [
+      Tv2ActionSubtype.DVE_LAYOUT,
+      Tv2ActionSubtype.DVE_INSERT_SOURCE_TO_INPUT,
+      Tv2ActionSubtype.RECALL_DVE,
+      Tv2ActionSubtype.DVE_INSERT_LAST_VIDEO_CLIP_TO_INPUT
+    ].includes(actionSubtype)
```

Comment:
The array looks to contain all the values of the used enum. 
Why not just get the enum values as a `string[]`, straight from the enum instead of creating an array with them?

Reply:
They won't continue to use all values of the enum. Right now it's just DVE that has added entries to it.
Your graphics Actions will also add entries to this enum.